arm-postgresql/2017-04-30-preview/swagger/postgresql.json

```diff
@@ -896,8 +896,21 @@
     },
     "ServerPropertiesForCreate": {
       "discriminator": "createMode",
-      "required": [ "createMode" ],
+      "required": [
+        "createMode",
+        "administratorLogin",
+        "administratorLoginPassword"
+      ],
       "properties": {
+        "administratorLogin": {
+          "type": "string",
+          "description": "The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation)."
+        },
+        "administratorLoginPassword": {
+          "type": "string",
+          "format": "password",
+          "description": "The password of the administrator login."
```

Comment:
@balajikris, this change is about issue https://github.com/Azure/azure-rest-api-specs/issues/1309# (the go generator doesn't support discriminator), I don't think it is a good way to change the swagger spec as a workaround, since with this change, it will impact other SDKs which support discriminator. Any idea about the balance of swagger specification and autorest generators?  Let's wait for autorest go generator to fix it? or have to remove the usage of discriminator?